app/services/publish/metadata_transfer_service.rb

```diff
@@ -30,6 +30,7 @@ def publish
 
     # @raise [Dor::DataError]
     def transfer_metadata(release_tags)
+      transfer_to_document_store(PublicCocinaService.create(item), 'cocina')
```

Comment:
I'd add a file extension for developer happiness; I think the current lack of extensions pre-dated purl-the-application (or otherwise is an artifact of previous system design goals).

```suggestion
      transfer_to_document_store(PublicCocinaService.create(item), 'cocina.json')
```

Reply:
I've added it.  Thanks for the suggestion.